CHANGES/511.bugfix

```diff
@@ -0,0 +1 @@
+Taught mongo-connection how to deal with Really Big Queries.
```

Comment:
This should be a user-oriented message in the changelog, any error message which user will no longer see or anything which a user can relate to. This message sounds too implementation detail oriented to me.